lib/player.test.js

```diff
@@ -164,10 +232,23 @@ describe("Player methods work correctly", () => {
       expect(result).toEqual(expected);
     });
 
-    it("getCpuGridChoice returns a grid choice", async () => {
-      const store = new Store();
+    it("getHumanPlayer gets human player from game", () => {
+      const player = new HumanPlayer();
+      const cpu = new CpuPlayer();
+      const game = new Game();
+      game.players = {
+        O: player,
+        X: cpu,
+      };
+      const expected = player;
+      const result = cpu.getHumanPlayer(game);
+      expect(result).toEqual(expected);
```

Comment:
I think the `expected = player` assignment just adds mental overhead without really simplifying anything
```suggestion
      expect(result).toEqual(player);
```
if `expected` was a complex object then I might agree with you more. 

I would even be comfortable with `expect(cpu.getHumanPlayer(game)).toEqual(player);` (or however prettier decides to format it) but that's more of a grey area.